atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/StrategySweeper.java

```diff
@@ -23,7 +23,7 @@
 import com.palantir.atlasdb.keyvalue.api.Value;
 import com.palantir.common.base.ClosableIterator;
 
-public interface SweepStrategySweeper {
+public interface StrategySweeper {
```

Comment:
Can you think of an even better name? We already have `SweepStrategy` so `SweeperStrategy` is wayyy too close to that for my liking. (But I agree that both `SweepStrategySweeper` and `StrategySweeper` are also awful)